From 73f2966dd3e7b47f9520d9451780e043f4ea90d1 Mon Sep 17 00:00:00 2001
From: root <root@jenkins2.schmoozecom.net>
Date: Mon, 9 Jun 2014 16:05:23 +0000
Subject: [PATCH 1/1] blacklist.sample

---
 blacklist.sample |   12 +++++++++++-
 1 files changed, 11 insertions(+), 1 deletions(-)

diff --git a/blacklist.sample b/blacklist.sample
index f054e28..411178a 100644
--- a/blacklist.sample
+++ b/blacklist.sample
@@ -14,10 +14,20 @@ blacklist wctc4xxp
 blacklist wcb4xxp
 blacklist wcaxx
 blacklist wcte43x
+blacklist tor3e
+blacklist opvxd115
+blacklist opvxa24xx
+blacklist opvxa1200
+blacklist zaphfc
+blacklist r1t1
+blacklist rxt1
+blacklist rcbfx
+blacklist xpp_usb
+
 
 # Some mISDN drivers may try to attach to cards supported by DAHDI. If you
 # have a card which is *not* supported by DAHDI but supported by one of the
 # below drivers you should feel free to remove it from the blacklist below.
 blacklist hfcmulti
 blacklist netjet
-blacklist hfcpci
+blacklist hfcpci
\ No newline at end of file
-- 
1.7.1

