From 6735495acffcd88a4ea1b7bc50e416a27463bef4 Mon Sep 17 00:00:00 2001
From: Jenkins Continuous Build server <jenkins@jenkins2.schmoozecom.net>
Date: Sun, 17 Aug 2014 03:17:58 +0000
Subject: [PATCH] Fixing modules.sample patch

---
 modules.sample |   25 +++++++++++++++++++++++++
 1 files changed, 25 insertions(+), 0 deletions(-)

diff --git a/modules.sample b/modules.sample
index 08692af..b320b5e 100644
--- a/modules.sample
+++ b/modules.sample
@@ -19,6 +19,10 @@ wct4xxp
 # Digium TE236
 wcte43x
 
+# OpenVox D115P/DE115P/D130P/DE130P: PCI Single-port T1/E1/J1 
+# OpenVox D115E/DE115E/D130E/DE130E: PCI-Express Single-port T1/E1/J1 
+#opvxd115 
+
 # Digium TE120P: PCI single-port T1/E1/J1
 # Digium TE121: PCI-Express single-port T1/E1/J1
 # Digium TE122: PCI single-port T1/E1/J1
@@ -37,6 +41,15 @@ wct1xxp
 # Digium TE110P: PCI single-port T1/E1/J1
 wcte11xp
 
+# Allo 1,2 and 4 port T1/E1/J1
+#tor3e
+
+# Rhino Single Span T1/E1/J1
+r1t1
+
+# Rhino Multi Span T1/E1/J1
+rxt1
+
 # Digium TDM2400P/AEX2400: up to 24 analog ports
 # Digium TDM800P/AEX800: up to 8 analog ports
 # Digium TDM410P/AEX410: up to 4 analog ports
@@ -52,6 +65,18 @@ wcfxo
 # Digium TDM400P: up to 4 analog ports
 wctdm
 
+# OpenVox A2410P: up to 24 analog ports
+#opvxa24xx
+
+# OpenVox A2410P: up to 12 analog ports
+#opvxa1200
+
+# Rhino FXO
+rcbfx
+
+# OpenVox B100P: 1 NT/TE BRI ports
+#zaphfc
+
 # Digium B410P: 4 NT/TE BRI ports
 wcb4xxp
 
-- 
1.7.1

